src/poc/server/poc-server/src/controllers/users.ts

```diff
@@ -86,4 +86,5 @@ export const emitConnection = async (req: Request, res: Response) => {
     const { uuid } = req.body;
     io.to(uuid).emit("event");
     res.status(200).json({ message: "Connection established" });
+    console.log("Connection established for", req.body)
```

Comment:
these changes needed?

Reply:
This is just to see if the api call worked, the other two have console messages.